Remotion/ObjectBinding/Validation.UnitTests/BusinessObjectValidationResultTest.cs

```diff
@@ -474,8 +486,8 @@ public void GetUnhandledValidationFailures_ForAllObjects_ReturnsFilteredResult (
 
       var unhandledValidationFailures = businessObjectValidationResult.GetUnhandledValidationFailures();
 
-      var unhandledValidationFailuresForResult1 = validationResult1.Errors.OfType<PropertyValidationFailure>()
-          .Where(f => f.ValidatedProperty.Equals(_firstNameProperty) || f.ValidatedProperty.Equals(_lastNameProperty));
+      var unhandledValidationFailuresForResult1 = validationResult1.Errors
+          .Where(f => f.ValidatedProperties.Single().Property.Equals(_firstNameProperty) || f.ValidatedProperties.Single().Property.Equals(_lastNameProperty));
```

Comment:
```suggestion
      var unhandledValidationFailuresForResult1 = validationResult1.Errors
          .Where(f => f.ValidatedProperties.Select(vp => vp.Property).All(p => p.Equals(_firstNameProperty) || p.Equals(_lastNameProperty });
```

I think this should work